src/common/config_opts.h

```diff
@@ -450,6 +450,8 @@ OPTION(mds_snap_min_uid, OPT_U32, 0) // The minimum UID required to create a sna
 OPTION(mds_snap_max_uid, OPT_U32, 65536) // The maximum UID allowed to create a snapshot
 OPTION(mds_snap_rstat, OPT_BOOL, false) // enable/disbale nested stat for snapshot
 OPTION(mds_verify_backtrace, OPT_U32, 1)
+// detect clients which aren't trimming completed requests
+OPTION(mds_max_completed_requests, OPT_U32, 100000)
```

Comment:
On Tue, Apr 21, 2015 at 2:35 AM, Gregory Farnum <notifications@github.com>
wrote:

> In src/common/config_opts.h
> <https://github.com/ceph/ceph/pull/4386#discussion_r28715940>:
>
> > @@ -450,6 +450,8 @@ OPTION(mds_snap_min_uid, OPT_U32, 0) // The minimum UID required to create a sna
> >  OPTION(mds_snap_max_uid, OPT_U32, 65536) // The maximum UID allowed to create a snapshot
> >  OPTION(mds_snap_rstat, OPT_BOOL, false) // enable/disbale nested stat for snapshot
> >  OPTION(mds_verify_backtrace, OPT_U32, 1)
> > +// detect clients which aren't trimming completed requests
> > +OPTION(mds_max_completed_requests, OPT_U32, 100000)
>
> This is a very large value, how'd you come up with it?
>

not so large I have a request blocked for > 30 seconds



> --
> Reply to this email directly or view it on GitHub
> <https://github.com/ceph/ceph/pull/4386/files#r28715940>.
>